src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -360,9 +379,8 @@ def check_created_partition_table(targetMap, installation)
             _(
               "Warning: There is no partition mounted as /boot.\n" +
                 "To boot from your hard disk, a small /boot partition\n" +
-                "(approx. %1) is required.  Consider creating one.\n" +
-                "Partitions assigned to /boot will automatically be changed to\n" +
-                "type 0x41 PReP/CHRP.\n" +
```

Comment:
Hmm, I'd start with @jsrain then `docu` team. It's probably not documented at all.